<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ShortcutsPage</class>
 <widget class="QWidget" name="ShortcutsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Action:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="actionNameLabel">
        <property name="text">
         <string>None</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Shortcuts:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="KeyCaptureLineEdit" name="keySeqLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string>clear</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="restoreShortcutsButton">
       <property name="text">
        <string>Restore Default Shortcuts</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KeyCaptureLineEdit</class>
   <extends>QLineEdit</extends>
   <header>keycapturelineedit.h</header>
   <slots>
    <signal>keyCaptured(QKeySequence)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>treeView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>ShortcutsPage</receiver>
   <slot>tableItemClicked(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>179</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>keySeqLineEdit</sender>
   <signal>keyCaptured(QKeySequence)</signal>
   <receiver>ShortcutsPage</receiver>
   <slot>keyCapLineEditTextChanged(QKeySequence)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restoreShortcutsButton</sender>
   <signal>clicked()</signal>
   <receiver>ShortcutsPage</receiver>
   <slot>restoreShortcutsButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>ShortcutsPage</receiver>
   <slot>clearButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tableItemClicked(QModelIndex)</slot>
  <slot>keyCapLineEditTextChanged(QKeySequence)</slot>
  <slot>restoreShortcutsButtonClicked()</slot>
  <slot>clearButtonClicked()</slot>
 </slots>
</ui>
